src/codemodder/scripts/generate_docs.py

```diff
@@ -138,6 +138,10 @@ class DocMetadata:
         importance="Medium",
         guidance_explained="Our change fixes explicitly insecure session configuration for a Flask application. However, there may be valid cases to use these insecure configurations, such as for testing or backward compatibility.",
     ),
+    "file-resource-leak": DocMetadata(
```

Comment:
It seems like most of our codemod names are framed in terms of the fix and not the problem. A simple suggestion would be `fix-file-resource-leak` but I'm open to suggestions.